projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```diff
@@ -117,6 +141,12 @@
     }
   }
 
+  .clr-vertical-nav {
```

Comment:
These styles are leaking to the vertical nav component. Please use a class name specific to the date picker. You can put two classes on the element such as `class="clr-vertical-nav clr-date-range-picker-nav"`.